greenplumpython/func.py

```diff
@@ -319,10 +342,12 @@ def transition_function(self) -> NormalFunction:
         ), f'Transition function of the aggregate function "{self.qualified_name}" is unknown.'
         return self._transition_func
 
-    def _create_in_db(self, db: Database) -> None:
+    def create_in_db(self, db: Database) -> None:
+        if self._transition_func is None:  # Function has already existed.
```

Comment:
why we not use self.transition_function here since it is a property.
and a little confuse about the `comment` here.

Reply:
copy that, thanks.